editaveis/tap.tex

```diff
@@ -0,0 +1,184 @@
+\section{Termo de Abertura do Projeto}
+Este documento tem como objetivo a formalização do projeto Unidade Móvel de
+Identificação de Saúde e Socorro (UMISS). As informações contidas nos tópicos 
+a seguir foram produzidos a fim de mostrar um resumo dos objetivos, riscos, 
+limites e recursos, bem como mostrar o estudo de viabilidade do projeto. 
+Tendo em vista as limitações de prazo, dinheiro e infraestrutura dos 
+participantes do projeto.
+\subsection{Descrição do Projeto}
+Para ter uma visão clara do projeto, foi usado o mapeamento de atividade no 
+modelo 5W2H, que está descrito a seguir:
+
+\begin{itemize}
+    \item \textbf{What:} Sitema integrado com subsistemas de movimentação,
+    monitoramento e estrutura. Desenvolvendo uma cadeira com possibilidade
+    de movimentação e captação dos sinais enviados pelo paciente. E monitoramento
+    através de um aplicativo nativo android e de uma plataforma web.
+    \item \textbf{Why:} Movimentar um indivíduo com mobilidade reduzida
+    e enviar respostas recolhidas
+    dos sensores para aplicativos e plataformas web dos seus respectivos
+    cuidadores.
+    \item \textbf{Where:} Em qualquer ambiente que tenha acesso a uma rede wifi.
+    \item \textbf{When:} Durante o primeiro semestre de 2017.
+    \item \textbf{Who:} Alunos dos cursos de Engenharias do Campus UnB Gama, nos
+    quais estão cursando a matéria de Projeto Integrador 2.
+    \item \textbf{How:} Utilizando a orientação dos professores da Universidade
+    de Brasília.
+    \item \textbf{How Much:} Cerca de R\$ 3.000,00, quanto a equipamento, por
+    cadeira.
+\end{itemize}
+
+\subsection{Propósito e justificativa do Projeto}
+O projeto tem como objetivo oferecer à pessoas indivíduo com mobilidade reduzida
```

Comment:
O início soou estranho. Sugestão: "O projeto tem como objetivo oferecer à familiares de pacientes com mobilidade reduzida uma cadeira de rodas motorizada, com fácil..."